modules/sync/admin-sync.php

```diff
@@ -127,14 +142,30 @@ protected function get_fields_to_sync( $post ) {
 
 		$postarr = parent::get_fields_to_sync( $post );
 
-		// For new drafts, save the date now otherwise it is overridden by WP. Thanks to JoryHogeveen. See #32.
-		if ( in_array( 'post_date', $this->options['sync'] ) && isset( $GLOBALS['pagenow'], $_GET['from_post'], $_GET['new_lang'] ) && 'post-new.php' === $GLOBALS['pagenow'] ) {
-			check_admin_referer( 'new-post-translation' );
+		if ( isset( $GLOBALS['post_type'] ) ) {
+			$post_type = $GLOBALS['post_type'];
+		} elseif ( isset( $_REQUEST['post_type'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			// 2nd case for quick edit.
+			$post_type = sanitize_key( $_REQUEST['post_type'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		}
+
+		// Make sure not to impact media translations when creating them at the same time as post
+		if ( in_array( 'post_parent', $this->options['sync'], true ) && ( ! isset( $post_type ) || $post_type !== $post->post_type ) ) {
+			unset( $postarr['post_parent'] );
+		}
+
+		$context_data = $this->check_context( (array) $post, false );
```

Comment:
In what cases `$_REQUEST['post_type']` isn't set?

Reply:
Seeing [this](https://github.com/polylang/polylang/blob/dcdd8e55295d95be185b2305524cae0cfc05586c/admin/admin-links.php#L100-L112), in none. This is the only place where the nonce is created, so the pos type should always be set. In that case there is no reason to not check the post type.

Reply:
Changed in [b46f995](https://github.com/polylang/polylang/pull/1532/commits/b46f9958e3ecf95f93599dbc87d873c8c5471465).